dask_snowflake/tests/test_core.py

```diff
@@ -214,3 +219,29 @@ def test_execute_params(table, connection_kwargs, client):
         check_dtype=False,
         check_index=False,
     )
+
+
+@pytest.mark.parametrize(
+    "partitioning",
+    [{"partition_size": "2 MiB"}, {"npartitions": 3}],
+    ids=["partition_size", "npartitions"],
+)
+def test_result_batching(partitioning, table, connection_kwargs, client):
+    ddf = (
+        dask.datasets.timeseries(freq="10s", seed=1)
+        .reset_index(drop=True)
+        .rename(columns=lambda c: c.upper())
+    )
+
+    to_snowflake(ddf, name=table, connection_kwargs=connection_kwargs)
+
+    ddf_out = read_snowflake(
+        f"SELECT * FROM {table}",
+        connection_kwargs=connection_kwargs,
+        **partitioning,
+    )
+
+    # Relatively loose check since we don't control batch sizes and the
+    # dask-snowflake batching algo is approximate.
+    assert ddf_out.npartitions < 6 and ddf_out.npartitions > 1
+    dd.utils.assert_eq(ddf, ddf_out, check_dtype=False, check_index=False)
```

Comment:
Parametrizing here seems off to me.  I'd rather use `partition_size` and check that the size was somewhere near close, and then try npartitions and check that the number of partitions was somewhere near close.  

Thoughts?

Reply:
Sure, that seems fine to me.